nservicebus/recoverability/subscribing-to-error-notifications.md

```diff
@@ -18,6 +18,13 @@ Error notifications are available for several events:
  * When a [delayed retry](/nservicebus/recoverability/#delayed-retries) occurs.
  * When a message fails all retries and is forwarded to the error queue.
 
+## Satellite behavior
+
+Since Version 6 errors happening in satellites like the [Gateway](/nservicebus/gateway/) and [TimeoutManager](/nservicebus/messaging/timeout-manager) will not trigger error notification and requires a [custom retry policy configuration per satellite](/nservicebus/satellites/#implementing-a-satellite-managing-errors). The gateway has its own [custom gateway retry policy](/nservicebus/gateway/#using-the-gateway-recoverability) API.
```

Comment:
>and requires a custom retry policy configuration per satellite

This isn't relevant for users right? They can only do something if we have chosen to expose an api, like we did for the GW? But eg. for the timeoutmanager they can no longer subscribe?

Reply:
mmm, seems like it. Now that I'm looking at that doco that seems only possible to do in the context of the feature.